AI_ML/CLASSIFICATION/SUPPORT_VECTOR_MACHINE/SUPPORT_VECTOR_MACHINE.py

```diff
@@ -1,14 +1,17 @@
-from flojoy import flojoy, DataContainer
+from flojoy import flojoy, DataFrame, Matrix
 import pandas as pd
 import numpy as np
 from sklearn import svm, preprocessing
-from typing import cast
+from typing import cast, Literal, Optional
 
 
 @flojoy
 def SUPPORT_VECTOR_MACHINE(
-    dc_inputs: list[DataContainer], params: dict
-) -> DataContainer:
+    training_data: DataFrame | Matrix,
+    input_data: DataFrame | Matrix,
+    target: Optional[str] = None,
+    kernel: Literal["linear", "poly", "rvm", "sigmoid"] = "linear",
```

Comment:
we should probably use all Literal values from scikit package for kernel parameter like: `kernel : {'linear', 'poly', 'rbf', 'sigmoid', 'precomputed'}` 
Also 'rvm' seems to be incorrect as this is not listed in SVC class docstring.

Reply:
I would even include a link to the SKLearn doc for SVM such that users understand what 'poly' or 'rbf' means.

Reply:
@Roulbac @smahmed776 Do I have to follow the docs convention which uses `rbf` as default or can I stick to `linear` as the default?

Reply:
It's ok to stick with `linear`, as long as studio users can read the doc and understand what the other possible values mean